### PR TITLE
Remove scheduling of database backups from documentation

### PR DESCRIPTION
**Pull request by dehawkins512**
Closes https://github.com/ManageIQ/manageiq.org/issues/1102

### CODE REVIEW

**Comment by dehawkins512** [comment]
@miq-bot assign @jrafanie

**Comment by dehawkins512** [comment]
@miq-bot unassign @jrafanie

**Comment by dehawkins512** [comment]
@miq-bot add_reviewer @jrafanie

File: general_configuration/_topics/configuration.md [hunk 1]
```diff
@@ -1568,153 +1566,6 @@ To schedule a SmartState Analysis or Compliance Check:
...
-3.  Set the `wal_keep_segments` value
-
-        postgres=# ALTER SYSTEM SET wal_keep_segments = <value>;
-
-4.  Reload `wal_keep_segments`
-
-        postgres=# SELECT pg_reload_conf();
-
-On successful backup, revise the value for `wal_keep_segments` to save
-space based on transaction log size.
```
Comment (jrafanie) [inline general_configuration/_topics/configuration.md:1604]: Note, the above information on `wal_keep_segments` seems related to database backups and not schedules of these backups.  @dehawkins512 do we have documentation on doing database/postgres backups using pg basebackup or pg_dump?  I wonder if this section should move there.

File: general_configuration/_topics/configuration.md [hunk 2]
```diff
@@ -1568,153 +1566,6 @@ To schedule a SmartState Analysis or Compliance Check:
...
-
-    If you change the **Time Zone**, you will need to reset the stating
-    date and time.
-
-10. Type or select a date to begin the schedule in **Starting Date**.
-
-11. Select a **Starting Time** (UTC) based on a 24 hour clock in the
-    selected time zone.
-
-12. Click **Add**.
```
Comment (jrafanie) [inline general_configuration/_topics/configuration.md:1656]: This line and above look good to remove.

File: general_configuration/_topics/configuration.md [hunk 3]
```diff
@@ -1568,153 +1566,6 @@ To schedule a SmartState Analysis or Compliance Check:
...
-
-  - Report
-
-  - Policy
-
-  - Alert
-
-  - SmartState Analysis
-
-  - Database Backup
```
Comment (jrafanie) [inline general_configuration/_topics/configuration.md:1712]: I think this is the only line that should be removed in this last section.

Comment (jrafanie) [reply general_configuration/_topics/configuration.md:1712]: The test of this from line 1658 to the end is about generically import/export of schedules, of which DB backup is one of them. We removed db backup schedules but the others still exist.

File: general_configuration/_topics/configuration.md (hunk 1 above)
Comment (dehawkins512) [reply general_configuration/_topics/configuration.md:1604]: @jrafanie Backing up and Restoring a database is documented here:
https://www.manageiq.org/docs/reference/latest/general_configuration/#backing-up-and-restoring-a-database-for-an-appliance-based-environment
The only docs that mention pg_dump are in the Install on Kubernetes and migrating from appliances topics here:
https://www.manageiq.org/docs/reference/morphy/installing_on_kubernetes/index.html

Comment (dehawkins512) [reply general_configuration/_topics/configuration.md:1604]: @jrafanie Can we confirm the information on `wal_keep_segments` is used in any DB backup other than a scheduled one?
I don't see where the ManageIQ docs mention using pg basebackup or pg_dump.

Comment (jrafanie) [reply general_configuration/_topics/configuration.md:1604]: @dehawkins512 I did some quick googling and I think it's no longer in postgres 13.  I guess we can remove it.  I think a replacment concept of `wal_keep_size` was added.  See: https://www.depesz.com/2020/07/27/waiting-for-postgresql-14-rename-wal_keep_segments-to-wal_keep_size/

See also: https://www.postgresql.org/docs/13/runtime-config-replication.html
(wal_keep_size added, wal_keep_segments removed)
vs:

https://www.postgresql.org/docs/12/runtime-config-replication.html
(still had wal_keep_segments)

**Review by jrafanie — APPROVED** [review]
LGTM

**Comment by dehawkins512** [comment]
@miq-bot add_labels petrosian/yes? oparin/yes? najdorf/yes? quinteros/yes?

**Comment by dehawkins512** [comment]
@miq-bot add_labels petrosian/yes?, oparin/yes?, najdorf/yes?, quinteros/yes?

**Comment by Fryguy** [comment]
Skipping backport to `quinteros`, because it is already in the branch.

**Comment by Fryguy** [comment]
Backported to `najdorf` in commit ccb0c3115a5327d974d4ec25046daa1ecd1bac9a.

```text
commit ccb0c3115a5327d974d4ec25046daa1ecd1bac9a
Author: Joe Rafaniello <jrafanie@users.noreply.github.com>
Date:   Tue Jul 11 14:14:07 2023 -0400

    Merge pull request #1735 from dehawkins512/dan_1102
    
    Remove scheduling of database backups from documentation
    
    (cherry picked from commit 29b1ced1c982479510e07def54bb4efbef6abd45)
```

**Comment by Fryguy** [comment]
Backported to `oparin` in commit 9678bf1b68b33752082887e34174c36755256229.

```text
commit 9678bf1b68b33752082887e34174c36755256229
Author: Joe Rafaniello <jrafanie@users.noreply.github.com>
Date:   Tue Jul 11 14:14:07 2023 -0400

    Merge pull request #1735 from dehawkins512/dan_1102
    
    Remove scheduling of database backups from documentation
    
    (cherry picked from commit 29b1ced1c982479510e07def54bb4efbef6abd45)
```

**Comment by Fryguy** [comment]
Backported to `petrosian` in commit 2d2ecb5a64dd76ae1e52d31fc282fcb7738b71c7.

```text
commit 2d2ecb5a64dd76ae1e52d31fc282fcb7738b71c7
Author: Joe Rafaniello <jrafanie@users.noreply.github.com>
Date:   Tue Jul 11 14:14:07 2023 -0400

    Merge pull request #1735 from dehawkins512/dan_1102
    
    Remove scheduling of database backups from documentation
    
    (cherry picked from commit 29b1ced1c982479510e07def54bb4efbef6abd45)
```

**Pull request merged**